res/xml/settings.xml

```diff
@@ -46,5 +46,8 @@
     <juloo.keyboard2.prefs.SlideBarPreference android:key="character_size" android:title="@string/pref_character_size_title" android:summary="@string/pref_character_size_summary" android:defaultValue="1.0" min="0.75" max="1.5"/>
     <juloo.keyboard2.prefs.IntSlideBarPreference android:key="key_vertical_space" android:title="@string/pref_key_vertical_space" android:summary="%sdp" android:defaultValue="2" min="0" max="8"/>
     <juloo.keyboard2.prefs.IntSlideBarPreference android:key="key_horizontal_space" android:title="@string/pref_key_horizontal_space" android:summary="%sdp" android:defaultValue="2" min="0" max="8"/>
+    <CheckBoxPreference android:key="border_config" android:title="@string/pref_border_config" android:summary="@string/pref_border_config_summary" android:defaultValue="false"/>
+    <juloo.keyboard2.prefs.IntSlideBarPreference android:key="border_radius" android:title="@string/pref_corners_radius" android:summary="%sdp" android:summaryOff="@string/pref_border_options_disabled" android:defaultValue="0" min="0" max="15" android:dependency="border_config"/>
+    <juloo.keyboard2.prefs.IntSlideBarPreference android:key="border_line_size" android:title="@string/pref_border_width" android:summaryOff="@string/pref_border_options_disabled" android:summary="%sdp" android:defaultValue="0" min="0" max="5" android:dependency="border_config"/>
```

Comment:
`1dp` is a lot, perhaps this option should be a `juloo.keyboard2.prefs.SlideBarPreference` that works with floating point numbers.